docs/tutorials/add-contracts-pallet.md

```diff
@@ -383,9 +379,7 @@ impl_runtime_apis! {
    /* --snip-- */
 
    /*** Add This Block ***/
-    impl contracts_rpc_runtime_api::ContractsApi<Block, AccountId, Balance, BlockNumber>
-		for Runtime
-	{
+	impl pallet_contracts_rpc_runtime_api::ContractsApi<Block, AccountId, Balance, BlockNumber> for Runtime {
```

Comment:
ContractExecResult::Success is changed in rc6, so that, line number 393,  Ok(v) => ContractExecResult::Success {
					status: v.status,
					data: v.data,
				}, must be changed too.